tests/getset.rs

```diff
@@ -0,0 +1,121 @@
+#[macro_use]
+extern crate getset;
+
+use crate::submodule::other::{Generic, Plain, Where};
+
+// For testing `pub(super)`
+mod submodule {
+    // For testing `pub(in super::other)`
+    pub mod other {
+        #[derive(MutGetters, Getters, Default)]
+        #[getset(get_mut)]
+        pub struct Plain {
+            /// A doc comment.
+            /// Multiple lines, even.
+            private_accessible: usize,
+
+            /// A doc comment.
+            #[getset(get = "pub", get_mut = "pub")]
+            public_accessible: usize,
+            // /// A doc comment.
+            // #[get_mut = "pub(crate)"]
+            // crate_accessible: usize,
+
+            // /// A doc comment.
+            // #[get_mut = "pub(super)"]
+            // super_accessible: usize,
+
+            // /// A doc comment.
+            // #[get_mut = "pub(in super::other)"]
+            // scope_accessible: usize,
```

Comment:
Could you please explain what these comments are for?

Reply:
They were, long ago, part of a future feature. Feel free to remove it or leave it in :)